services/streammanager/kafka/client/producer.go

```diff
@@ -36,7 +36,7 @@ type Producer struct {
 }
 
 // NewProducer instantiates a new producer. To use it asynchronously just do "go p.Publish(ctx, msgs)".
-func (c *Client) NewProducer(topic string, producerConf ProducerConfig, skipTopicInitialisation bool) (p *Producer, err error) { // skipcq: CRT-P0003
+func (c *Client) NewProducer(topic string, producerConf ProducerConfig) (p *Producer, err error) { // skipcq: CRT-P0003
```

Comment:
we can get rid of topic params as per @fracasula suggestion

Reply:
Addressed